docs/Tutorials/Visualization.md

```diff
@@ -2,26 +2,81 @@
 Distributed under the MIT License.
 See LICENSE.txt for details.
 \endcond
-# Visualization {#tutorial_visualization}
+# Running and Visualizing {#tutorial_visualization}
 
-### 1D %Data Volume %Data
+### Building an Executable from an Existing Source File
 
-Volume data from one dimensional simulations is visualized using the python
-script `Support/bin/Render1dData.py`. Run
-`python /path/to/Render1dData.py --help` to view the help text for the script.
-Let us consider a Burgers evolution where we want to visualize the solution,
-`U`, then in the run directory you can use
+SpECTRE source files for executables are located in
```

Comment:
for evolution executables (there are others in Executables, and soon in Elliptic/Executables)